packages/eslint-plugin/src/utils/is-compiled-import.ts

```diff
@@ -0,0 +1,32 @@
+import type { Rule, Scope } from 'eslint';
```

Comment:
yea, there is duplicated code between isCss and isKeyFrame. It's good that you refactored them. 👍